format/spec.md

```diff
@@ -193,16 +204,14 @@ Supported primitive types are defined in the table below. Primitive types added
 
 Notes:
 
-1. Decimal scale is fixed and cannot be changed by schema evolution. Precision can only be widened.
```

Comment:
Is this being removed because we intend to change it in V3?  Maybe this should be included in the summary?

Reply:
This is cleanup to make things more consistent.